Applications/Viqui/vqCore.cxx

```diff
@@ -3387,6 +3396,43 @@ void vqCore::generateReport(QString path, bool generateVideo)
   this->exportKml(path);
 }
 
+//-----------------------------------------------------------------------------
+void vqCore::exportIqr()
+{
+  this->updateIqrModel();
+
+  QString path =
+    vgFileDialog::getSaveFileName(
+      nullptr, "Save IQR model...", QString(), "IQR File (*.iqr);;");
+
+  if (path.isEmpty())
+    {
+    return;
+    }
+
+  QFile file(path);
+  if (!file.open(QIODevice::WriteOnly |
```

Comment:
...would prefer to omit the line break here; it isn't needed and I'm not really feeling that it's beneficial either.